fk/secretreceive.go

```diff
@@ -206,13 +234,77 @@ func countDigits(i int) (count int) {
 	return len(iString)
 }
 
+func getFileNameToWrite(filename string) (string, error) {
```

Comment:
This would be clearer if it were called e.g. `getAbsoluteFilenameInDownloadsDir(basename string)`